CLAUDE.md

```
@@ -47,6 +47,30 @@ mix ash.codegen <resource_name>   # Generate resource code
 # Variables in .env files override all config defaults
 ```
 
+## Database Configuration
+
+### IMPORTANT: Test Environment Database Setup
+
+The test environment uses SQL Sandbox pool for safe concurrent testing. The configuration is:
+
+**Key Points:**
+- Test environment MUST use `Ecto.Adapters.SQL.Sandbox` pool
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)